source/services/GoogleAnalytics.jsx

```diff
@@ -1,8 +1,14 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 import ReactGA from 'react-ga';
 import { Route } from 'react-router-dom';
 
+type props = {
+  location: {
```

Comment:
I believe React Router should have a predefined type for the location prop. Check out https://github.com/HackIllinois/site-2019/blob/staging/flow-typed/npm/react-router-dom_v4.x.x.js#L51